src/TagValidator.php

```diff
@@ -18,7 +18,7 @@ class TagValidator extends AbstractValidator
      */
     protected $rules = [
         'name' => ['required'],
-        'slug' => ['required', 'unique:tags'],
+        'slug' => ['required', 'unique:tags', 'regex:/^[^\/ ]*$/i'],
```

Comment:
Backslashes create the same issue

```suggestion
        'slug' => ['required', 'unique:tags', 'regex:/^[^\/\\ ]*$/i'],
```

Reply:
Good catch